src/olympia/ratings/serializers.py

```diff
@@ -44,6 +45,9 @@ def get_addon(self, obj):
             'slug': addon.slug
         }
 
+    def get_is_developer_reply(self, obj):
```

Comment:
observation: a property that could be on the `Rating` model itself.  Not that it'd give us anything extra, really.

Reply:
Yeah, I've looked through the code and we wouldn't really make use of that property anywhere. Generally we care more about the fact that a rating has a reply than the fact that it *is* a reply.